plugins/out_cloudwatch_logs/cloudwatch_api.c

```diff
@@ -546,6 +697,22 @@ int process_and_send(struct flb_cloudwatch *ctx, struct cw_flush *buf,
     int found = FLB_FALSE;
     struct flb_time tms;
 
+    /* Added for EMF support */
+    struct flb_ir_metric *metric;
+
+    int ir_metric_type;
+    char *ir_metric_unit;
+    if (strncmp(input_plugin, "cpu", INPUT_PLUGIN_CPU_LENGTH) == 0) {
```

Comment:
nit: since you have the literal string "cpu" here, not sure how useful it is to have a separate constant for the length of the string... probably more readable just to put the number 3 here

Reply:
Updated.